modules/install/join_rke2_agent.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 # This script is used to join one or more nodes as agents
 echo "$@"
-# the following lines are to enable debug mode
-set -x
```

Comment:
my pov is that is no longer needed since we are now moving to a good shape of "info" available, since we are already printing/echoing usefull stuff and also errors . but let me know

Reply:
yeah I'm totally fine without it, just wanted to make sure it was intentional!